### ISSUE TITLE
create set-up support for torrent web ui on DuckieTV GitHub Wiki

### ISSUE THREAD

**Issue by garfield69** [SchizoDuckie/DuckieTV#527]
- [x] Add a link to qBittorrent32plus.js, pointing to the existing DuckieTV GitHub Wiki qBt32+ support page.
- [x] Create a DuckieTV GitHub Wiki page for setting up Vuze WebUi.
- [x] Change the vuze.html line 6 link to point to this. 
- [x] Create a DuckieTV GitHub Wiki page for setting up Deluge WebUi.
- [x] Add a link to the deluge.html to point to this. 
- [x] Create a DuckieTV GitHub Wiki page for setting up qBitTorrent (pre 3.2) WebUi.
- [x] Add a link to the qBittorrent.html to point to this. 
- [x] Create a DuckieTV GitHub Wiki page for setting up Transmission WebUi.
- [x] Add a link to the translission.html to point to this. 
- [x] Create a DuckieTV GitHub Wiki page for setting up &micro;Torrent WebUi.
- [x] Add a link to the utorrentwebui.html to point to this. 
- [x] Create a DuckieTV GitHub Wiki page for setting up tixati WebUi.
- [x] Add a link to the tixati.html to point to this.

**Comment by garfield69** [comment]
Hmm. I am unable to establish a connection between Vuze and DuckieTV.
I have installed Win7 Vuze 5.6.2.0 and initially installed the azureus HTML Web UI 0.7.7 plugin, and set up Vuze settings to use http, port 6886 (default), and username and password. 
On DuckieTV's Vuze Integration I set up the same port and username and password, but I am getting 404.
Since then I have replaced azureus HTML Web UI 0.7.7 with the Beta 1.0.2, but that has not made any difference.
My Vuze HTML Web UI setttings:
![image](https://cloud.githubusercontent.com/assets/6933240/9240167/4afd8750-41b7-11e5-9149-d41d43842780.png)
My DuckieTV Vuze Integration settings and console log:
![image](https://cloud.githubusercontent.com/assets/6933240/9240184/7ca7b988-41b7-11e5-9deb-facf4f78d43b.png)
I've restarted both apps several times, and tried various combinations, no joy.

Has anyone got the interface working with Vuze at all?
Anyone got suggestions for debugging this?

**Comment by garfield69** [comment]
Interesting. I can access vuze web ui from a chrome session using my public ip address
![image](https://cloud.githubusercontent.com/assets/6933240/9243653/3c35c478-41e7-11e5-8004-562798fa66f8.png)
and by using my local address
![image](https://cloud.githubusercontent.com/assets/6933240/9243661/5ece6a58-41e7-11e5-96f7-ac39e1b71ae4.png)
which suggests to me the problem might be with the vuze (transmission) api ?

**Comment by garfield69** [comment]
All the Web UI connections have been tested and documented, except Vuze, which I cannot as yet, get going.

**Comment by garfield69** [comment]
Ah. looks like windows Vuze is no longer using the same protocol as transmission any more?
By setting in vuze.js line 25
```javascript
            rpc: ''
```
I now connect and start getting responses. However, the replies are not what the transmissionAPI is expecting.
I do not know if Vuse on the other platforms has changed similarly, but there is not reason to expect otherwise.

**Comment by garfield69** [comment]
OK. Problem solved. \o/

line 6 in templates/settings/vuze.html used to read:
```javascript
    <p>see <a href='https://wiki.vuze.com/w/Setup_WebUI_without_GUI' target='_blank'>here</a>
 for setup instructions (outdated, install the 'Azureus WebUI plugin' from the application) </p>
```
When I looked up the plugins list in the app, the closest match I found was:
`Azureus HTML WebUI`
However this plugin implements a completely different method to what DuckieTV was set up to use.

Turns out that the plugin I should I installed is called:
`Vuze Web Remote`
Once I installed this, the DuckieTV Vuze Web UI Interface worked like a Charm. :-D
I will now update the wiki.
